server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/pipelines_controller.rb

```diff
@@ -17,7 +17,8 @@
 module ApiV1
   module Admin
     class PipelinesController < ApiV1::BaseController
-      before_action :check_pipeline_group_admin_user_and_401
+      before_action :check_pipeline_group_admin_user_and_401, except: [:show]
```

Comment:
I think you are confused between pipeline instance that we display for view users and pipeline config. No one apart from pipeline-group-admin and super-admin are allowed to view/edit/create pipelines. However, based on the configuration, some underprivileged users may have view/operate access to pipeline instances. They are two totally different things. What you originally had ie. `before_action :check_pipeline_group_admin_user_and_401` was the right thing to do. The same goes for the other changes you have made for this to work. Please fix.